packages/js/create-woo-extension/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0.0](https://www.npmjs.com/package/@woocommerce/create-woo-extension/v/1.0.0) - 2022-12-15
+
+-   Patch - Add WC validation [#35947]
+
+[See legacy changelogs for previous versions](https://github.com/woocommerce/woocommerce/blob/68581955106947918d2b17607a01bdfdf22288a9/packages/js/create-woo-extension/CHANGELOG.md).
```

Comment:
Perhaps we should have a separate `changelog.txt` to keep a running list of previous changelogs and only keep the most recent in `CHANGELOG.MD`? Like how we do with Woo core?

Reply:
We could, thats a monorepo changelog tool change though and would affect all JS packages